appendices/issue-review-template.md

```diff
@@ -0,0 +1,22 @@
+```markdown
```

Comment:
@leouieda @ocefpaf @xmnlab @yuvipanda @lheagy @xmnlab one. more item to review. This is short. it's the submission template where we collect information about the package before review. if you have time to look at it and provide any feedback please do.